css/bulma-rtl.css

```diff
@@ -6839,7 +6821,7 @@ label.panel-block:hover {
 
 .tabs.is-boxed a {
   border: 1px solid transparent;
-  border-radius: 0 0 4px 4px;
```

Comment:
Bug introduced in rtl version since border-radius is top-l top-r bottom-r bottom-l there was no need to change this property
![image](https://user-images.githubusercontent.com/6115458/84577824-ec1e7800-adbf-11ea-9444-b22f1cee90c7.png)